src/main/native/include/sysid/view/Analyzer.h

```diff
@@ -51,6 +51,7 @@ class Analyzer : public glass::View {
   void PrepareGraphs();
   void RefreshInformation();
   void AbortDataPrep();
+  void LoadFeedForwardGains(bool combined = false);
```

Comment:
Can you rename this to `DisplayFeedforwardGains()`? Feedforward is also one word.